app/Config/App.php

```diff
@@ -442,4 +442,23 @@ class App extends BaseConfig
      * @see http://www.w3.org/TR/CSP/
      */
     public bool $CSPEnabled = false;
+
+
+    /**
+     * --------------------------------------------------------------------------
+     * Cross-Origin Resource Sharing (CORS)
+     * --------------------------------------------------------------------------
+     *  To pass cors needs to enter valid url in corsValidDomains variable then
+     *  go to Route.php add this line $routes->setAutoRoute(true).
+     *
+     *  this step is necessary to  pass cors in rest ful api , Auto Route at Route.php must
+     *  be true and uncomment.
```

Comment:
Turning on auto routing puts your apps at risk.
It is not recommended to turn it on as the default setting for CI4 is off. 
You should manually set the necessary routes.